core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -289,7 +289,7 @@ sealed class Vec[T <: Data] private[chisel3] (gen: => T, val length: Int) extend
 
   /** Creates a dynamically indexed read or write accessor into the array.
     */
-  override def apply(p: UInt): T = macro CompileOptionsTransform.pArg
+  override def apply(x: UInt): T = macro CompileOptionsTransform.oneArg
```

Comment:
```suggestion
  override def apply(p: UInt): T = macro CompileOptionsTransform.oneArg
```
Please don't change argument names as that is a source incompatible change